.circleci/cimodel/data/simple/binary_smoketest.py

```diff
@@ -72,7 +72,7 @@ def gen_tree(self):
         is_master_only=True,
     ),
     # This job has an average run time of 3 hours o.O
-    # Now only running this on master to reduce overhead
+    # Now only running this on main to reduce overhead
```

Comment:
I see there is a "is_master_only" variable in this file. Will it require renaming?

Reply:
Yes. I can either rename it now to be `is_master_or_main_only` then update it to `is_main_only` when I clean up all the dual support after the change or just do the second and leave this as is for now. I think it's still accurate until the flip of branches.

Reply:
it's fine, don't bother renaming now